phpunit.xml.dist

```diff
@@ -146,6 +146,10 @@
 		<file>tests/Backend/Workspaces/WorkspacesRenameLoadTest.php</file>
 		<directory>tests/Backend/Synapse</directory>
 		<file>tests/Backend/Export/ExportParamsTest.php</file>
+
+		<!--FILE WORKSPACE-->
+		<file>tests/Backend/FileWorkspace/WorkspacesLoadTest.php</file>
+		<file>tests/Backend/FileWorkspace/WorkspacesUnloadTest.php</file>
```

Comment:
Prosímtě tohle šíbování mezi suitama je proč? Optimalizace času jednotlivých testů?

Reply:
JJ ta part3 teď protabuje build a part2 končí brzo takže sem to prohodil.